packages-backend/loggers/src/loggers.spec.ts

```diff
@@ -85,18 +111,12 @@ describe('application logger', () => {
         authorization: 'Bearer 123',
       },
     });
-    // @ts-ignore
-    expect(console._stdout.write).toHaveBeenCalledWith(
-      expect.stringContaining(
-        `\"meta\":${JSON.stringify({
-          req: {
-            headers: {
-              authorization: '[REDACTED]',
-            },
-            headersJsonString: JSON.stringify({ authorization: '[REDACTED]' }),
-          },
-        })}`
-      )
-    );
+    // Note that authorization is REDACTED in both headers and headersJsonString
+    expect(console._stdout.write.mock.lastCall).toMatchInlineSnapshot(`
```

Comment:
Just migrated the expectation too after making sure the old one worked with my changes made.